File: Assignment___41/Program3.c
//_______________________________________________________________________________________________________________________________
//
//  File name :     Program3.c
//
//  Description :   Write a recurive program which display below pattern.
// 
//                  Input  :  5
//
//                  Output :  5  4  3  2  1
//
//  Author :        Sarika Devidas Gadhe
//
//_______________________________________________________________________________________________________________________________




#include<stdio.h>


void Display(int iNo)
{
    static int i = 0;
    static int iNum = 5;

    if(i < iNo)
    {
        printf("%d\t",iNum);
        i++;
        iNum--;
        Display(iNo);
    }
   
}

int main()
{
    int iValue = 0;

    printf("Enter number : \n");
    scanf("%d",&iValue);

    Display(iValue);

    printf("\n");
    return 0 ;
}

File: Assignment___49/Program3.java
//_______________________________________________________________________________________________________________________________
//
//  File name :     Program3.java
//
//  Description :   Write a program which accept String from user and display below pattern.
//                  
//                  Input :     Hello
//                  
//                  Output :    
//                              H   #   #   #   #     
//                              H   e   #   #   #  
//                              H   e   l   #   #    
//                              H   e   l   l   #     
//                              H   e   l   l   o     
//                                   
//
//  Author :        Sarika Devidas Gadhe
//
//_______________________________________________________________________________________________________________________________


import java.util.*;

class Pattern
{
    public void Pattern(String str)
    {
        int i = 0, j = 0;

        for(i = 1; i <= str.length(); i++)
        {
            for(j = 0; j < str.length() ; j++)
            {
                if(i > j)
                {
                    System.out.print(str.charAt(j)+"\t");
                }  
                else
                {
                    System.out.print("#\t");
                }
            }
            System.out.println();
        }
       
    }
}

class Program3
{
    public static void main(String A[])
    {
        Scanner sobj = new Scanner(System.in);

        System.out.println("Enter String : ");
        String str = sobj.nextLine();
       

        Pattern pobj = new Pattern();
        pobj.Pattern(str);

        sobj.close();


    }
}